metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_core.py

```diff
@@ -264,14 +278,12 @@ def extract_dbt_entities(
             "ephemeral",
             "test",
         ]:
-            logger.debug(f"Loading schema info for {dbtNode.dbt_name}")
-            if catalog_node is not None:
-                # We already have done the reporting for catalog_node being None above.
-                dbtNode.columns = get_columns(
-                    catalog_node,
-                    manifest_node,
-                    tag_prefix,
-                )
+            dbtNode.columns = get_columns(
+                dbtNode.dbt_name,
+                catalog_node,
```

Comment:
Earlier, we checked if catalog_node is empty, but now we don't check. 
Was this on purpose?